src/server-code/admin-server.js

```diff
@@ -19,6 +19,8 @@ import { exec } from 'child_process';
 // Helps us parse post requests from falcor
 import bodyParser from 'body-parser';
 
+import request from 'request';
```

Comment:
We don't currently have this module as a dependency, I think the only reason that it's probably working is because some of our dependencies have it as dependencies. I know this has a slightly nicer API than the built-in `http` module, so I guess we could maybe use it, what's your opinion? Is it a big enough improvement over `http` to warrant the extra module? It's also only a serverside module which means we're not sending the code over the internet so that's a pro on the list of using it.

So yeah, if we do end up using it you should add it as a dependency

Reply:
oops I must have forgot. will add